pkg/intermediate/aggregate.go

```diff
@@ -366,6 +366,16 @@ func (a *AggregationProcess) addOrUpdateRecordInMap(flowKey *FlowKey, record ent
 				}
 			}
 		} else {
+			// For inter-node traffic with deny np or drop anp, we may receive the record from the conntrack table first.
+			// In this case, the record from the conntrack table will be considered to do correlation job and the ReadyToSend
+			// is false until we receive another record from PacketIn from another node. However, the record From PacketIN
+			// is not correlationRequired, so we won't send this kind of connection due to this conflict.
```

Comment:
I don't fully understand what's going on and I'll probably defer to @heanlan and @dreamtalen, but I do not like that the comment is so specific to the Antrea FE implementation, with mentions of conntrack and PacketIn. While I know that this code is pretty specific to Antrea, this seems *too* specific.

Reply:
I've updated the comment. Thanks.